docker-compose.yml

```diff
@@ -1,5 +1,49 @@
 version: "3"
 services:
+  celestia-devnet:
+    container_name: celestia_devnet
+    image: ghcr.io/rollkit/local-celestia-devnet:v0.11.0-rc12
+    restart: on-failure
+    ports:
+      - "26657"
+      - "26658"
+      - "26659"
+      - "9090"
+
+  evm:
+    container_name: evm_base_shard
+    environment:
+      - DA_AUTH_TOKEN=${DA_AUTH_TOKEN:-}
+      - VALIDATOR_NAME=${VALIDATOR_NAME:-}
+      - CHAIN_ID=${CHAIN_ID-}
+      - KEY_NAME=${KEY_NAME:-}
+      - KEY_MNEMONIC=${KEY_MNEMONIC:-}
+      - KEY_BACKEND=${KEY_BACKEND:-}
+      - TOKEN_AMOUNT=${TOKEN_AMOUNT:-}
+      - STAKING_AMOUNT=${STAKING_AMOUNT:-}
+      - MIN_GAS_PRICE=${MIN_GAS_PRICE:-}
+      - TOKEN_DENOM=${TOKEN_DENOM:-}
+      - FAUCET_ADDR=${FAUCET_ADDR:-}
+      - DA_BASE_URL=${DA_BASE_URL:-}
+      - DA_BLOCK_HEIGHT=${DA_BLOCK_HEIGHT:-}
+      - BLOCK_TIME=${BLOCK_TIME:-}
+      - DA_NAMESPACE_ID=${DA_NAMESPACE_ID:-}
+      - DA_CONFIG=${DA_CONFIG:-}
+    image: us-docker.pkg.dev/argus-labs/world-engine/chain:latest
+    expose:
+      - "1317"
+      - "26657"
+      - "9090"
+      - "9601"
+    ports:
+      - "1317:1317"
+      - "26657:26657"
+      - "9090:9090"
+      - "9601:9601"
+      - "8545:8545"
+    depends_on:
```

Comment:
hm, i'm tempted to not make this depends on celestia devnet because that would create coupling issues when we want to deploy to prod (i.e. have to run celestia devnet although we dont need to)

Reply:
sure, will remove